internal/install/static_snapshot_yml.go

```diff
@@ -89,10 +89,12 @@ services:
     environment:
     - "FLEET_ENROLL=1"
     - "FLEET_ENROLL_INSECURE=1"
+    - "FLEET_SERVER_INSECURE_HTTP=1"
     - "FLEET_INSECURE=1"
```

Comment:
You need to add `FLEET_SERVER_HOST=0.0.0.0`. This will allow an Elastic Agent running on your machine to enroll into this fleet-server.

Reply:
Ok, added this one too.